### PR TITLE
Add bubble columns

### PR DESCRIPTION
**Pull request by WoofWoofDoggo**
This PR adds bubble columns. They look like water (with moving bubble particles), and can push you in any direction with a configurable strength.

Requested by me

### CODE REVIEW

File: Entities/BubblePushField.cs [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            }
+
+            foreach (WindMover mover in Scene.Tracker.GetComponents<WindMover>()) { 
+                if(mover.Entity.CollideCheck(this)) {
+                    if (WindMovers.ContainsKey(mover))
+                        WindMovers[mover] = Calc.Approach(WindMovers[mover], Strength, Engine.DeltaTime / .6f);
+                    else
+                        WindMovers.Add(mover, 0f);
+                } else {
+                    if(WindMovers.ContainsKey(mover)) {
```
Comment (0x0ade) [inline Entities/BubblePushField.cs]:
```suggestion
                    if (WindMovers.ContainsKey(mover)) {
```

File: Entities/BubblePushField.cs [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+using System;
+using System.Collections.Generic;
+using Monocle;
+
+namespace Celeste.Mod.SpringCollab2020.Entities {
+    [CustomEntity("SpringCollab2020/bubblePushField")]
+    class BubblePushField : Entity {
+        public new int Width;
+
+        public new int Height;
```
Comment (0x0ade) [inline Entities/BubblePushField.cs]: This kiiinda begs for issues...

File: Entities/BubblePushField.cs [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+        public BubbleParticle(bool active, bool visible) : base(active, visible) { }
+
+        public override void Added(Entity entity) {
+            base.Added(entity);
+
+            BubbleField = (BubblePushField) entity;
+            Position = BubbleField.Position;
+
+            Rand = BubbleField.Rand;
+            Texture = GFX.Game["particles/SpringCollab2020/bubble_" + new string[] { "a", "b" }[Rand.Next(0, 1)]];
```
Comment (0x0ade) [inline Entities/BubblePushField.cs]: Array should be stored in a static readonly field.

**Comment by WoofWoofDoggo** [comment]
@Cruor @0x0ade 
Should be good to go now, I think.

**Pull request merged**